action.yml

```
@@ -20,6 +24,9 @@ runs:
           gh pr view --json labels -q '.labels[]|.name' ${REPOSITORY_URL}/pull/$pr_number | while read label ; do
             if [[ -z "$labels" ]] || [[ ",$labels," =~ ",$label," ]]; then
               gh pr edit --add-label "$label" $MERGE_QUEUE_PR_URL
-            if
+            fi
+          done
+          for additional-label in $(echo $additional-labels | sed "s/,/ /g") ; do
```

Can we set the env with`ADDITIONAL_LABELS: ${{ additional-labels }}` and:

```suggestion
          for additional-label in $(echo $ADDITIONAL_LABELS | sed "s/,/ /g") ; do
```